### DISCUSSION TITLE
Cannot use `make` to build sdb library [Question]

### DISCUSSION THREAD

**Discussion by alexpanter** [mono/sdb#80] [Questions] [answered]
Hi,
In the README.md you write:

First, clone the submodules:
$ git submodule update --init --recursive
To build, run:
$ make

I have problems running the make - command prompt says the make command doesn't exist.
Perhaps I am missing something fundamental, would be appreciated if someone could help me so I can use the debuggin tools.
Thanks in advance,
Alexander

**Comment by tritao** [comment]
Which platform are you using?

**Comment by alexpanter** [comment]
I am running Windows 10, if that is what you are asking :)

**Comment by tritao** [comment, marked as answer]
OK then you need to install a POSIX environment like https://www.cygwin.com/ to be able to build it.

**Comment by alexrp** [comment]
Closing as the question has been answered.
